events/entry.go

```diff
@@ -40,11 +48,59 @@ var getDockerClient = func() (*docker.Client, error) {
 	return NewDockerClient(false)
 }
 
-var getHandlers = func(dockerClient *docker.Client) map[string]Handler {
-	handler := &StartHandler{
+var getHandlers = func(dockerClient *docker.Client) (map[string]Handler, error) {
```

Comment:
I've thought about this and our conversation and you're right. There are better, cleaner ways to accomplish what I want here.

Will fix.